federation/apis/federation/v1beta1/generated.proto

```diff
@@ -89,6 +89,8 @@ message ClusterSpec {
   // This is to help clients reach servers in the most network-efficient way possible.
   // Clients can use the appropriate server address as per the CIDR that they match.
   // In case of multiple matches, clients should use the longest matching CIDR.
+  // +patchMergeKey=clientCIDR
+  // +patchStrategy=merge
```

Comment:
Should these get copied here? I guess it doesn't hurt anything. The tags should really be stripped out by the proto generator, I guess that can be fixed in another PR.